### ISSUE TITLE
mIou

### ISSUE THREAD

**Issue by sriharsha0806** [brianhuang1019/FCN-pytorch#1]
Can you also mention about mIoU? how much did you acheive?

**Comment by AlainOUYANG** [comment]
Hello, for a limited time I run the code with 20 epoch and I got 0.17745 for the meanIU and 71.54% for pixel accuracy. What about you ?

**Comment by brianhuang1019** [comment]
Which dataset did you run

**Comment by brianhuang1019** [comment]
IoU is about 0.65~0.70

**Comment by BXuan694** [comment]
> IoU is about 0.65~0.70

some of IoUs are ~0.60, however, some are ~0.1, thus mIoU would be ~0.2. Does that make sense?

**Comment by yaopengzero** [comment]
I got a miou=0.53 ,and pixel Accurancy=0.897 in CamVid dataset with training 500 epoches

**Comment by taronegeorage** [comment]
> I got a miou=0.53 ,and pixel Accurancy=0.897 in CamVid dataset with training 500 epoches

btw, did you change the parameters? Same accuracy, but mean IoU is only 0.25 within 500 epoches.

**Comment by Water2style** [comment]
> I got a miou=0.53 ,and pixel Accurancy=0.897 in CamVid dataset with training 500 epoches

can you share your configs？
i train about 100 epochs,and the miou is abot 25%,
i use CamVid and set the num_class to 32,is this the reason ?

**Comment by Dong-qs** [comment]
I trained 500 epochs on Cityscapes dataset. However, the accuracy was just 89%,and iou is 46%. How to improve them? Thank you very much!
